.github/workflows/deploy_web_demo.yml

```diff
@@ -39,7 +39,7 @@ jobs:
         with:
           profile: minimal
           target: wasm32-unknown-unknown
-          toolchain: 1.84.0
+          toolchain: 1.85.0
```

Comment:
Do we still need 1.85? If so, I can update `egui main` so the diff of this PR is smaller 🤷